oshdb-util/src/main/java/org/heigit/bigspatialdata/oshdb/util/changesetenricher/OSMChangeset.java

```diff
@@ -0,0 +1,170 @@
+package org.heigit.bigspatialdata.oshdb.util.changesetenricher;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import org.heigit.bigspatialdata.oshdb.util.OSHDBBoundingBox;
+import org.heigit.bigspatialdata.oshdb.util.OSHDBTimestamp;
+
+public class OSMChangeset {
+  private final long id;
+  private final Long user_id;
```

Comment:
should be the primitive type long and not Long Object type!

Reply:
For now we use this tool: https://github.com/ToeBee/ChangesetMD that clearly states:
> Note that all fields except for id and created_at can be null.

Therefore we have to use Objects, I guess